### DISCUSSION TITLE
Unable to install telescope via packer nvim

### DISCUSSION THREAD

**Discussion by ghost** [wbthomason/packer.nvim#890] [Q&A]
When I install it  I get the following error : 

![image](https://user-images.githubusercontent.com/104435563/165296993-da4d997a-e6aa-44fe-904c-717acbd67cf3.png)

More info : 

![image](https://user-images.githubusercontent.com/104435563/165297207-de8deda0-bc14-4e01-b142-81b3cbc2a1f5.png)

![image](https://user-images.githubusercontent.com/104435563/165297274-5219fbfd-b496-4d19-889c-17111fd4fd1c.png)

![image](https://user-images.githubusercontent.com/104435563/165297353-bf98834b-9105-4899-a853-506f09395319.png)

![image](https://user-images.githubusercontent.com/104435563/165297701-84cfec6e-a073-4532-a46e-f20f1d7e9e38.png)

**Comment by bryant-the-coder** [comment]
`requires = "nvim-lua/plenary.nvim"`

**Reply by ghost** [reply]
Hi, Its the same result. In addition im copy pasting from the telescope page, so it should work by default

![image](https://user-images.githubusercontent.com/104435563/165305496-80996650-bcaf-4aec-a98b-33cea579959d.png)
